packages/engine/src/database/service.ts

```diff
@@ -76,6 +77,19 @@ export class DatabaseService extends Service {
   }
 
   async destroy() {
+    if (yn(process.env.DATABASE_TRANSIENT)) {
+      try {
+        const trx = await this.knex.transaction()
+        for (const table of this.tables.reverse()) {
+          await trx.schema.dropTable(table.id)
+          this.logger.debug(`Dropped table '${table.id}'`)
+        }
+        await trx.commit()
+      } catch (e) {
+        this.logger.error(e, 'Failed to destroy transient database')
+      }
+    }
```

Comment:
Here's a new setting. You can set your DB to be transient so it's cleared when the server closes.

Reply:
Nice! But, I never know, if there is an error during the commit call, is rollback automatically called or we should rather call it explicitly in the catch block?

Reply:
I know that changes aren't applied unless commit is called, but not calling rollback explicitly might cause the connection to fail to close. I'll add the rollback in the catch block